CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 ### Unreleased
 
 - Execute flag `mergeErrors` allows inline errors to facilitate returning multiple errors from resolvers that use `execute`.
+- Imported resolvers proxy as an `execute` call to imported component and type resolvers are no longer pulled up.
```

Comment:
https://www.npmjs.com/package/graphql-component - I'm seeing 1.2.3 as the latest version. Did something go wrong publishing recently?

Reply:
Probably forgot to update changelog properly.

Reply:
I’m not so sure, looks like latest version of package.json on master is set to 1.2.2

Reply:
Fixed.